improver/calibration/dz_rescaling.py

```diff
@@ -219,8 +220,13 @@ def _create_hour_coord(self, source_cube: Cube, target_cube: Cube):
             target_cube: Cube to which an auxiliary coordinate will be added.
         """
         coord_name = "forecast_reference_time"
```

Comment:
This is only used once now, so could just be defined in the `long_name=` line below.

Reply:
I've modified this as suggested.